pkg/rehearse/jobs.go

```diff
@@ -230,19 +225,31 @@ func NewExecutor(toBeRehearsed map[string][]prowconfig.Presubmit, prNumber int,
 	}
 }
 
+func printAsYaml(pjs []*pjapi.ProwJob) error {
+	sort.Slice(pjs, func(a, b int) bool { return pjs[a].Spec.Job < pjs[b].Spec.Job })
+	jobAsYAML, err := yaml.Marshal(pjs)
+	if err == nil {
+		fmt.Printf("%s\n", jobAsYAML)
+	}
+
+	return err
+}
+
 // ExecuteJobs takes configs for a set of jobs which should be "rehearsed", and
 // creates the ProwJobs that perform the actual rehearsal. *Rehearsal* means
 // a "trial" execution of a Prow job configuration when the *job config* config
 // is changed, giving feedback to Prow config authors on how the changes of the
 // config would affect the "production" Prow jobs run on the actual target repos
 func (e *Executor) ExecuteJobs() (bool, error) {
 	submitSuccess := true
-	pjs, err := e.submitRehearsals()
+	names, pjs, err := e.submitRehearsals()
```

Comment:
If you change to just return the prowjob structs, can you do the reduction to a set of strings here rather than in the `submitRehearsals()`?

Reply:
Addressed.